api/internal/plugins/loader/loader.go

```diff
@@ -106,12 +106,16 @@ func relativePluginPath(id resid.ResId) string {
 		strings.ToLower(id.Kind))
 }
 
-func (l *Loader) AbsolutePluginPath(id resid.ResId) (string, error) {
+func (l *Loader) AbsolutePluginPath(id resid.ResId) ([]string, error) {
+	var plugins []string
 	pluginHome, err := l.absPluginHome()
 	if err != nil {
-		return "", err
+		return nil, err
+	}
+	for _, dir := range pluginHome {
+		plugins = append(plugins, filepath.Join(dir, relativePluginPath(id), id.Kind))
```

Comment:
Previously, this returned the one possible plugin location--if the plugin turned out not to be there, then it was missing. A plugin should not actually exist in two places, and in any case we can't use it from two places at once. So I think this should continue to return a single value, i.e. the first value that this loop finds that actually exists. Do you agree?

Reply:
This Loads PluginHomes from the function we're changing to return a []string (list of paths) whereas, earlier it was returning a string i.e an absolute path. Hence, I'd made this change. In order to return a single value here, we'll have to assume that line 149 will have only a single value, `dir[0]`, should we assume this?
```
// Check default locations for a valid plugin root, and cache it if found.                                                                                          
dir, err := konfig.ValidAbsPluginHomes(l.fs)
```